lib/apiservers/engine/backends/container_proxy.go

```diff
@@ -870,6 +871,56 @@ func (c *ContainerProxy) AttachStreams(ctx context.Context, vc *viccontainer.Vic
 	return nil
 }
 
+// Rename() calls the portlayer's containerRenamehandler to update the container name in the handle,
+// and then calls CommitHandler to commit the new name to vSphere
+func (c *ContainerProxy) Rename(vc *viccontainer.VicContainer, newName string) error {
```

Comment:
Can you make sure we return the same http error codes the docker specifies in their documentation? Same goes for `Container.Rename` above.